state.go

```diff
@@ -159,8 +159,8 @@ func (s state) validate() (bool, string) {
 // if the env variable is empty or unset, an empty string is returned
 // if the string does not start with '$', it is returned as is.
 func subsituteEnv(name string) string {
```

Comment:
Looks like this function essentially is the same thing as os.ExpandEnv so maybe just drop it completely?

Reply:
I was going to make a "A WILD $THING APPEARS" joke but I couldn't find one SFW... I updated the PR because why not?